src/verify.rs

```diff
@@ -73,7 +88,10 @@ fn compile_and_test(exercise: &Exercise, run_mode: RunMode) -> Result<bool, ()>
     progress_bar.finish_and_clear();
 
     match result {
-        Ok(_) => {
+        Ok(output) => {
+            if verbose {
+                println!("{}", output.stdout);
```

Comment:
I'm starting to think that we should be a bit more sophisticated while handling the output of exercises, and try to have both stdout and stderr merged as they would appear in the terminal screen.

Anyway, out of scope, just thinking out loud.

Reply:
@jrvidal thanks for the review! I agree with providing more information to the end users as long as there is a way to control that; like with command line options to make output `--quiet` or `--verbose`.

Normally, an exercise's standard error should be empty unless there is a complication error or a runtime one. Unless someone explicitly uses `eprint!` or `eprintln!` (or the many other ways of writing to targeted file handles) in their code to solve an exercise, there should be nothing in the standard error. So, keeping standard output and error separate may actually come in handy when trying to figure out if a user actually wrote to the standard error.

I am open to discussing this further, though.

Thanks!

Abdou

Reply:
> I'm starting to think that we should be a bit more sophisticated while handling the output of exercises, and try to have both stdout and stderr merged as they would appear in the terminal screen.
> 
> Anyway, out of scope, just thinking out loud.

Done in the `v6` branch with `os_pipe` :D